database/index.js

```diff
@@ -1,24 +1,73 @@
 const mysql = require('mysql');
+const util = require('util');
 const mysqlConfig = require('./config');
 
-class MySqlDatabase {
-  constructor(config) {
-    this.connection = mysql.createConnection(config);
+// create mysql connection
+const connection = mysql.createConnection(mysqlConfig);
+
+// promisify connection.query
+connection.query = util.promisify(connection.query);
+
+// helper database functions using async/await
+const insertRecord = async (tableName, recordObjs) => {
+  const queryOptions = {
+    sql: Array(recordObjs.length + 1).join(`INSERT INTO ${tableName} SET ?;`),
+    values: recordObjs,
+  };
+
+  try {
+    const data = connection.query(queryOptions);
+    return (await data).map(row => Object.assign({}, row));
+  } catch (err) {
+    throw new Error(`insert ${tableName} error: + ${err.message}`);
   }
+};
 
-  query(options) {
-    const context = this;
-    return new Promise((resolve, reject) => {
-      context.connection.query(options, (err, data, fields) => {
-        if (err) { reject(err); }
-        resolve(data, fields);
-      });
-    });
+const truncateAllTables = async () => {
+  try {
+    const data = connection.query(`SET FOREIGN_KEY_CHECKS = 0;
+      TRUNCATE users;
+      TRUNCATE rooms;
+      TRUNCATE reviews;
+      SET FOREIGN_KEY_CHECKS = 1;`);
+    return (await data).map(row => Object.assign({}, row));
+  } catch (err) {
+    throw new Error(`truncate error: + ${err.message}`);
   }
-}
+};
 
-const connection = new MySqlDatabase(mysqlConfig);
+const queryReviewsByRoomId = async (queryObj) => {
+  let sql = 'SELECT userName, avatar, date, (accuracy + communication + cleanliness + location + checkIn + value)/6 AS aggregateRate, text FROM ' +
```

Comment:
It would be better practice to use the length of an array containing the column names rather than hardcoding the number 6.